schedule/virt_autotest/kubevirt-tests.yaml

```diff
@@ -0,0 +1,38 @@
+name:           kubevirt-tests
+description:    >
+    Maintainer: nan.zhang@suse.com
+    Kubevirt server & agent node installation and test modules
+vars:
+    SCC_ADDONS: contm,sdk,legacy
+schedule:
+    - boot/boot_from_pxe
+    - installation/welcome
+    - installation/accept_license
+    - installation/scc_registration
+    - installation/addon_products_sle
+    - installation/system_role
+    - installation/partitioning
+    - installation/partitioning_firstdisk
+    - installation/partitioning_finish
+    - installation/installer_timezone
+    - installation/user_settings
+    - installation/user_settings_root
+    - installation/resolve_dependency_issues
+    - installation/installation_overview
+    - installation/disable_grub_timeout
+    - installation/disable_grub_graphics
+    - installation/start_install
+    - installation/await_install
+    - installation/logs_from_installation_system
+    - installation/reboot_after_installation
+    - virt_autotest/login_console
+    - virt_autotest/install_package
+    - virt_autotest/update_package
```

Comment:
Do you still need     
- virt_autotest/install_package
- virt_autotest/update_package
for kubevirt test ?

Reply:
Yes, it should be included as Alice suggested.